pageserver/src/virtual_file/owned_buffers_io/write/flush.rs

```diff
@@ -222,6 +224,8 @@ pub struct FlushBackgroundTask<Buf, W> {
     channel: duplex::mpsc::Duplex<FullSlice<Buf>, FlushRequest<Buf>>,
     /// A writter for persisting data to disk.
     writer: Arc<W>,
+    /// Gate guard is held on as long as we need to do operations in the path (delete on drop)
```

Comment:
**delete** on drop? What gets deleted on drop? Where?

This module is the wrong place to document what its consumers do.